eternalcore-core/src/main/java/com/eternalcode/core/feature/randomteleport/RandomTeleportSettingsImpl.java

```diff
@@ -0,0 +1,153 @@
+package com.eternalcode.core.feature.randomteleport;
+
+import com.eternalcode.core.delay.DelaySettings;
+import java.time.Duration;
+import java.util.EnumSet;
+import java.util.Set;
+import net.dzikoysk.cdn.entity.Contextual;
+import net.dzikoysk.cdn.entity.Description;
+import org.bukkit.Material;
+
+@Contextual
+public class RandomTeleportSettingsImpl implements RandomTeleportSettings, DelaySettings {
+    @Description({
+        "# Type of random teleportation,",
+        "# WORLD_BORDER_RADIUS - radius based on the world-border size.",
+        "# STATIC_RADIUS - radius based on the manually value."
+    })
+    public RandomTeleportType randomTeleportType = RandomTeleportType.WORLD_BORDER_RADIUS;
+
+    @Description({
+        "# Radius of random teleportation, this uses for starting point spawn via /setworldspawn.",
```

Comment:
Radius of random telefortation area centred at spawn point set via /setworldspawn